portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,16 +17,43 @@
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> quotes;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add(
+        "A ship in port is safe, but that is not what ships are for. "
+            + "Sail out to sea and do new things. - Grace Hopper");
+    quotes.add("They told me computers could only do arithmetic. - Grace Hopper");
+    quotes.add("A ship in port is safe, but that's not what ships are built for. - Grace Hopper");
+    quotes.add("It is much easier to apologise than it is to get permission. - Grace Hopper");
+    quotes.add("If you can't give me poetry, can't you give me poetical science? - Ada Lovelace");
+    quotes.add("I am in a charming state of confusion. - Ada Lovelace");
+    quotes.add(
+        "The Analytical Engine weaves algebraic patterns, "
+            + "just as the Jacquard loom weaves flowers and leaves. - Ada Lovelace");
+    quotes.add(
+        "Sometimes it is the people no one can imagine anything of "
+            + "who do the things no one can imagine. - Alan Turing");
+    quotes.add("Those who can imagine anything, can create the impossible. - Alan Turing");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String quote = quotes.get((int) (Math.random() * quotes.size()));
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Lee!</h1>");
```

Comment:
Is this `<h1>Hello Lee!</h1>` Supposed to be part of the response? Couldn't you move it to the HTML code and just return the quote here?

Reply:
You are right, the html code was also returned as part of the quote.